frontend_tests/node_tests/user_groups.js

```diff
@@ -0,0 +1,32 @@
+set_global('page_params', {
+    is_admin: false,
+    realm_users: [],
+});
+
+set_global('$', function () {
+});
+
+set_global('blueslip', {});
+
+zrequire('user_groups');
+
+(function test_user_groups() {
+    var admins = {
+        name: 'Admins',
+        id: 1,
+    };
+    var all = {
+        name: 'Everyone',
+        id: 2,
+    };
+    user_groups.add(admins);
+    assert(user_groups.get_user_group_from_id(admins.id));
```

Comment:
You could check here that the object returned by the method is the same as `admins`